modules/search/class.jetpack-search.php

```diff
@@ -153,12 +153,24 @@ public static function instance() {
 	 */
 	public function setup() {
 		if ( ! Jetpack::is_active() || ! Jetpack_Plan::supports( 'search' ) ) {
+			/**
+			 * Fires when the Jetpack Search fails and would fallback to MySQL.
+			 *
+			 * @since 7.9.0
+			 *
+			 * @param string $reason Reason for Search fallback.
+			 * @param mixed  $data   Data associated with the request, such as attempted search parameters.
+			 */
+			do_action( 'jetpack_search_abort', 'inactive', null );
 			return;
 		}
 
 		$this->jetpack_blog_id = Jetpack::get_option( 'id' );
 
 		if ( ! $this->jetpack_blog_id ) {
+			/** This action is documented in modules/search/class.jetpack-search.php
+			 */
```

Comment:
```suggestion
```

Reply:
Ah thanks! I started to do it multiline before checking the core guidebook.